src/content/test/errors/status-messages.tsx

```diff
@@ -0,0 +1,162 @@
+                <p>
+                <label for="email">Username (required): </label>
+                <input type="text" name="username" id="username"
+                </p>
+                <p>
+                <input type="submit" name="submit" id="submit" value="Submit">
+                </p>
+                </form>
+            `}
+        />
+
+        <h2>More examples</h2>
+
+        <h3>WCAG success criteria</h3>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html">
+                Understanding Success Criterion 4.1.3: Status Messages
+            </Markup.HyperLink>
+        </Markup.Links>
+
+        <h3>Sufficient techniques</h3>
+        <h4>If a status message conveys a suggestion, or a warning on the existence of an error:</h4>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA19">
+                Using ARIA role=alert or Live Regions to Identify Errors
+            </Markup.HyperLink>
+        </Markup.Links>
+        <p>in combination with any of the following:</p>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G83">
+                Providing text descriptions to identify required fields that were not completed
+            </Markup.HyperLink>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G84">
+                Providing a text description when the user provides information that is not in the list of allowed values
+            </Markup.HyperLink>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G85">
+                Providing a text description when user input falls outside the required format or values
+            </Markup.HyperLink>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G177">
+                Providing suggested correction text
+            </Markup.HyperLink>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G194">
+                Providing spell checking and suggestions for text input
+            </Markup.HyperLink>
+        </Markup.Links>
+
+        <p>If a status message advises on the success or results of an action, or the state of an application:</p>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22">
+                Using role=status to present status messages
+            </Markup.HyperLink>
+        </Markup.Links>
+        <p>in combination with:</p>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G199">
+                Providing success feedback when data is submitted successfully
+            </Markup.HyperLink>
+        </Markup.Links>
+
+        <p>If a status message conveys information on the progress of a process</p>
```

Comment:
The other similar messages in this section use trailing colons; should be consistent